### PR TITLE
Fixes some bugs

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
(that I introduced)

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!bugfix

**Comment by NonQueueingMatt** [comment]
!review

File: html/changelogs/mattatlas-bobthefixer.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ChangeMe
```
Comment (JohnWildkins) [inline html/changelogs/mattatlas-bobthefixer.yml]: hmm

Comment (NonQueueingMatt) [inline html/changelogs/mattatlas-bobthefixer.yml]:
```suggestion
author: MattAtlas
```

**Pull request merged**